Detectors/ITSMFT/MFT/calibration/src/NoiseCalibratorSpec.cxx

```diff
@@ -57,6 +59,7 @@ void NoiseCalibratorSpec::init(InitContext& ic)
   mPathDcs = ic.options().get<std::string>("path-DCS");
   mOutputType = ic.options().get<std::string>("send-to-server");
   mNoiseMapForDcs.clear();
+  api.init("http://alice-ccdb.cern.ch");
```

Comment:
Do not init unconditionally new ccdbApi instance if it is used only in some cases. Don't define corresponding data member api, when needed, create a local instance (see below).